docker/build.sh

```diff
@@ -57,19 +57,21 @@ SHELL ["bash","-c"]
 #
 # Additional python libs
 #
-RUN pip install nxpd graphviz pudb dask_labextension sphinx sphinx_rtd_theme recommonmark numpydoc $CUPY
-RUN conda install -y -c conda-forge python-graphviz bqplot=0.11.5 nodejs=11.11.0 jupyterlab=0.35.4 \
-    ipywidgets=7.4.2 pytables mkl numexpr
+RUN pip install nxpd dask_labextension $CUPY
```

Comment:
dask_labextension can be installed via conda too
https://anaconda.org/conda-forge/dask-labextension

Reply:
Sure! I’ll do. Somehow I missed that one! Thanks!

Reply:
Hi @yidong72,

Done! Could you please review it!

Thanks!!!